### PR TITLE
Erik's sample pull request

### PR DESCRIPTION
**Pull request by eriktrautman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ Pro tip - tropical trailmix is delicious in oatmeal  -- Abby
 
 Sunshine on my shoulders makes me happy! -- Driver Erik
 
+Dan is contributing to the project!!!! - Dan
 
+Dan, dan, motion man.  --Driver Erik
 
+THIS IS WHAT I WANT TO ADD!!!!! - Dan
+
+ERIK'S MERGE CONFLICT HAHAHAHAHA
```
Comment (dsberger) [inline README.md]: This is pretty mean.

File: README.md [hunk 2]
```diff
@@ -43,6 +43,11 @@ Pro tip - tropical trailmix is delicious in oatmeal  -- Abby
 
 Sunshine on my shoulders makes me happy! -- Driver Erik
 
+Dan is contributing to the project!!!! - Dan
```
Comment (dsberger) [inline README.md]: Wow I was very excited.

Comment (dakmau) [inline README.md]: Clearly the best Dan is here

File: README.md [hunk 3]
```diff
@@ -43,6 +43,16 @@ Pro tip - tropical trailmix is delicious in oatmeal  -- Abby
...
 
+Dan, dan, motion man.  --Driver Erik
 
+HELLO EVERYONE EH -- Driver Daniel
 
+THIS IS WHAT I WANT TO ADD!!!!! - Dan
```
Comment (dakmau) [inline README.md]: Hello everyone

File: README.md [hunk 4]
```diff
@@ -39,10 +39,21 @@ Erik will use Windows by the time I'm through
 
 Abhi wants a nickname.  Any suggestions? -- Erik the Driver
 
+This is the cOnFlIcTy COWBOY BOOT of the schwad - Schwad
```
Comment (Schwad) [inline README.md]: I appreciate incorporation of cowboy boots.

**Comment by Schwad** [comment]
Wow. Such high level comment

File: README.md [hunk 5]
```diff
@@ -16,7 +30,7 @@ You can always do whatever you want.  You are unique and special. - Erik
 
 What did you have for lunch today? - Driver Erik
 
-I had some prep-work goodness and a bit of trailmix. -Matt 
+I had some prep-work goodness and a bit of trailmix. -Matt
```
Comment (ayva) [inline README.md]: Olga's comment

**Comment by facingsouth** [comment]
test pull request

**Pull request merged**